torchaudio/metrics.py

```diff
@@ -0,0 +1,40 @@
+from collections.abc import Sequence
```

Comment:
Does this really deserve to be in a new namespace, for user-side API perspective?
I could also see that this can live inside of `torchaudio.functionals`.

Reply:
Splitting the implementation to the dedicated module makes sense. `functionals.py` grew too large in my opinion, but we can also import functions defined in this module in `functionals.py` so that users can access it via `torchaudio.functionals`

Reply:
That's a very good question. We'll need to follow-up on where this should go. `functionals.py` does seem like a good candidate.